packages/mask/src/utils/hooks/usePersonaProofs.ts

```diff
@@ -0,0 +1,38 @@
+import { EMPTY_LIST } from '@masknet/shared-base'
+import { NextIDProof } from '@masknet/web3-providers'
+import { useEffect } from 'react'
+import { useAsyncFn, useEffectOnce } from 'react-use'
+import { MaskMessages } from '../messages.js'
+
+export function usePersonaProofs(publicKey?: string) {
```

Comment:
A duplicate hook.

Reply:
The hook need use the `MaskMessage`. Not have common package.